pytorch3d/implicitron/dataset/dataset_base.py

```diff
@@ -144,6 +151,30 @@ def __getitem__(self, key):
     def __len__(self):
         return len(fields(self))
 
+    def crop_by_bbox_(self, box_crop_context):
+        clamp_bbox_xyxy = _clamp_box_to_image_bounds_and_round(
```

Comment:
Please add `FrameData.cropped` property and don’t allow cropping if the frame is already cropped, otherwise we can get rubbish.